fintech-examples/fintech-ui/src/app/bank/sidebar/sidebar.component.html

```diff
@@ -16,8 +16,8 @@
         <span class="nav__title">Accounts</span>
       </a>
     </li>
-    <li class="nav-item{{ showInitiatePayment ? '' : '--disabled' }} sidebar-element">
-      <a class="nav-link" [routerLink]="['payment']">
+    <li class="nav-item{{ showInitiatePayment && isLoaDone() ? '' : '--disabled' }} sidebar-element">
```

Comment:
a better solution could be usage of [ngClass](https://angular.io/api/common/NgClass) directive